src/openrct2/ride/coaster/meta/WaterCoaster.h

```diff
@@ -55,5 +55,11 @@ constexpr const RideTypeDescriptor WaterCoasterRTD =
     SET_FIELD(ColourKey, RideColourKey::Ride),
     SET_FIELD(Name, "water_coaster"),
     SET_FIELD(LightFXAddLightsMagicVehicle, LightFxAddLightsMagicVehicle_BoatHire),
+    SET_FIELD(StartRideMusic, OpenRCT2::RideAudio::DefaultStartRideMusicChannel),
+    SET_FIELD(DesignCreateMode, TrackDesignCreateMode::Default),
+    SET_FIELD(MusicUpdateFunction, DefaultMusicUpdate),
+    SET_FIELD(Classification, RideClassification::Ride),
+    SET_FIELD(UpdateLeaveEntrance, PeepUpdateRideLeaveEntranceDefault),
+    SET_FIELD(UpdateMeasurementsSpecialElements, RideUpdateMeasurementsSpecialElements_WaterCoaster),
```

Comment:
This seems like a lot more stuff than this PR says it does.

Reply:
This is necessary to set the values of defaulted fields that are before the UpdateMeasurements field in the rtd structure. It doesn't build if you don't set those.